### PR TITLE
feat(EMS-572): Policy and exports - Multiple contract policy - months of cover field

### PR DESCRIPTION
**Pull request by ttbarnes**
This PR adds "months of cover" options, field validation and data saving to the Multiple contract policy page.

## Summary of changes
- Change total months of cover field from a text input to a select drop down. Populate the month values dynamically via new mapping file - `mapTotalMonthsOfCover`.
- Add `totalMonthsOfCover` to the schema and DB dump.
- Define the maximum months of cover in constants, as per business requirements.
- Add validation handling for if the field is submitted without a value.

## Other improvements

- Rename `mapPeriodMonths` helper function to have a generic name, `mapMonthString`.
- Add form submission and "when going back to the page" E2E tests for the Multiple contract policy page.
- Add overall E2E validation test for empty form submission on the Multiple contract policy page.

### CODE REVIEW

File: e2e-tests/cypress/e2e/journeys/insurance/policy-and-exports/multiple-contract-policy/multiple-contract-policy.spec.js [hunk 1]
```diff
@@ -264,4 +285,47 @@ context('Insurance - Policy and exports - Multiple contract policy page - As an
...
+
+    describe('when going back to the page', () => {
+      it('should have the submitted values', () => {
+        goToPageDirectly(referenceNumber);
+
+        multipleContractPolicyPage[REQUESTED_START_DATE].dayInput().should('have.value', '1');
+        multipleContractPolicyPage[REQUESTED_START_DATE].monthInput().should('have.value', getMonth(startDate));
+        multipleContractPolicyPage[REQUESTED_START_DATE].yearInput().should('have.value', getYear(startDate));
+
+        multipleContractPolicyPage[TOTAL_MONTHS_OF_COVER].inputOptionSelected().contains('2');
```
Comment (Zainzzkk) [inline e2e-tests/cypress/e2e/journeys/insurance/policy-and-exports/multiple-contract-policy/multiple-contract-policy.spec.js:327]: is there are reason we are using contains and not equal?

Comment (ttbarnes) [reply e2e-tests/cypress/e2e/journeys/insurance/policy-and-exports/multiple-contract-policy/multiple-contract-policy.spec.js:327]: The rendered select options have "months" string - didn't see much value in adding this in here. Can add if you'd like?

Comment (Zainzzkk) [reply e2e-tests/cypress/e2e/journeys/insurance/policy-and-exports/multiple-contract-policy/multiple-contract-policy.spec.js:327]: I am just thinking if the months changed to say 23, this test will still pass

Comment (ttbarnes) [reply e2e-tests/cypress/e2e/journeys/insurance/policy-and-exports/multiple-contract-policy/multiple-contract-policy.spec.js:327]: Good point, i'll update it 👍

File: src/ui/server/helpers/data-content-mappings/map-month-string.ts [hunk 2]
```diff
@@ -1,15 +1,15 @@
 /**
- * mapPeriodMonths
+ * mapMonthString
  * Map credit period answer to a month/s string
  * @param {Number} Credit period
  * @returns {String} Credit period with month/s string
  */
-const mapPeriodMonths = (answer: number) => {
+const mapMonthString = (answer: number) => {
```
Comment (Zainzzkk) [inline src/ui/server/helpers/data-content-mappings/map-month-string.ts:7]: could answer ever be 0?

File: src/ui/server/helpers/mappings/map-total-months-of-insurance/index.ts [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import mapMonthString from '../../data-content-mappings/map-month-string';
+
+/**
+ * mapTotalMonthsOfCover
+ * Map all months of covers options and mark if it's selected
+ * @param {Array} Array of numbers
```
Comment (Zainzzkk) [inline src/ui/server/helpers/mappings/map-total-months-of-insurance/index.ts:6]: we're missing a param here?

File: src/ui/server/helpers/data-content-mappings/map-month-string.ts (hunk 2 above)
Comment (ttbarnes) [reply src/ui/server/helpers/data-content-mappings/map-month-string.ts:7]: Potentially, in which case it'd return `0 months`

File: src/ui/server/helpers/mappings/map-total-months-of-insurance/index.ts (hunk 3 above)
Comment (ttbarnes) [reply src/ui/server/helpers/mappings/map-total-months-of-insurance/index.ts:6]: Sorry, typo, good eye

**Review by Zainzzkk — APPROVED** [review]
LGTM
